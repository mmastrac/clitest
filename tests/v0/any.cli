#!/usr/bin/env clitest --v0

$ echo "a\nb\nc"
*

$ echo "a\nb\nc"
! a
*

$ echo "a\nb\nc"
! a
! b
*

$ echo "a\nb\nc"
! a
! b
! c
*

$ echo "a\nb\nc"
! a
*
! c

$ echo "a\nb\nc"
%EXPECT_FAILURE
! a
! b
! c
! d
*

$ echo "a\nb\nc\nd\ne"
! a
! b
*
! d
! e

$ echo "start\n1\n2\nend\nstart\n1\n2\nend"
repeat {
    ! start
    *
    ! end
}
