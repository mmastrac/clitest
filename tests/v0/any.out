Running <root>/any.cli ...
printf "a\nb\nc\n"
---
a
b
c
---
✅ OK

printf "a\nb\nc\n"
---
a
b
c
---
✅ OK

printf "a\nb\nc\n"
---
a
b
c
---
✅ OK

printf "a\nb\nc\n"
---
a
b
c
---
✅ OK

printf "a\nb\nc\n"
---
a
b
c
---
✅ OK

printf "a\nb\nc\n"
---
a
b
c
---
✅ OK

printf "a\nb\nc\nd\ne\n"
---
a
b
c
d
e
---
✅ OK

printf "start\n1\n2\nend\nstart\n1\n2\nend\n"
---
start
1
2
end
start
1
2
end
---
✅ OK

<root>/any.cli PASSED
